src/subdomains/generic/kyc/dto/kyc-error.enum.ts

```diff
@@ -61,6 +63,8 @@ export const KycErrorMap: Record<KycError, string> = {
   [KycError.RELEASED]: undefined,
   [KycError.RESTARTED_STEP]: undefined,
   [KycError.USER_DATA_DEACTIVATED]: 'Account deactivated',
+  [KycError.IDENT_IP_COUNTRY_MISMATCH]: 'Due to regulatory requirements',
+  [KycError.IP_COUNTRY_MISMATCH]: 'Due to regulatory requirements',
```

Comment:
Maybe something like `Regulatory requirements not met` so it fits the other errors (same for other languages)?